sample-app/src/components/SearchBar.tsx

```diff
@@ -25,9 +26,7 @@ export default function SearchBar({ placeholder, isVertical, screenReaderInstruc
   const isLoading = useAnswersState(state => state.searchStatus.isLoading);
 
   function executeQuery () {
-    isVertical
-      ? answersActions.executeVerticalQuery()
-      : answersActions.executeUniversalQuery();
+    executeSearchWithUserLocation(answersActions, isVertical, {}, true);
```

Comment:
This reads a bit funny. To a developer, it would seem as if we're always including the userLocation in the search. Additionally, the SDK's `SearchBar` allows you to configure the GeoLocation call. So, we should probably add `PositionProps` to `Props` here too.

Reply:
update: split into functions inside `SearchHandler` class as suggested in another comment